README.md

```diff
@@ -139,5 +139,5 @@ If you like this project, give me a star
 
 This project is [MIT](./MIT.md) licensed.
 
- <p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Actually, you have two README files, the one from the root folder and the other in the src folder. Kindly remove one. 🚮